pkg/ride/functions_proto.go

```diff
@@ -178,11 +178,6 @@ func performInvoke(invocation invocation, env environment, args ...rideType) (ri
 			Asset:     payment.Asset,
 		}
 	}
-	address, err := env.state().NewestRecipientToAddress(recipient)
```

Comment:
We need to have script addess be set before applying payments, before the function `getLibVersion` expects to see script address in ws.callee()
So, it's better not to move setting address from here